tests/unit/Codeception/Module/AssertsTest.php

```diff
@@ -66,8 +71,60 @@ public function testExceptionFails()
      */
     public function testOutputExceptionTimeWhenNothingCaught()
     {
-        $module = new \Codeception\Module\Asserts(make_container());
-        $module->expectException(RuntimeException::class, function () {
+        $this->module->expectException(RuntimeException::class, function () {
+        });
+    }
+
+    public function testExpectThrowable()
+    {
+        $this->module->expectThrowable('Exception', function () {
+            throw new Exception();
+        });
+        $this->module->expectThrowable(new Exception('here'), function () {
+            throw new Exception('here');
+        });
+        $this->module->expectThrowable(new Exception('here', 200), function () {
+            throw new Exception('here', 200);
+        });
+    }
+
+    /**
+     * @expectedException PHPUnit\Framework\AssertionFailedError
```

Comment:
Hey, could you update the code to not use this annotation? 
Because of https://github.com/sebastianbergmann/phpunit/issues/3332

Reply:
Yes, sure, thanks for pointing that out, didn't know that! :)

Fixed in f8a2b5289cc907bb095ee0be889d1fa86ef086af